tmt/steps/provision/mrack.py

```diff
@@ -1179,6 +1180,16 @@ def _create(self, tmt_name: str) -> None:
             public_key=self.public_key,
             group=self.beaker_job_group)
 
+        mrack_requirement = self.api._mrack_transformer.create_host_requirement(data)
```

Comment:
@skycastlelily should this not be under the below condition also? Does not seems to be used anywehre else?